plugins/org.jkiss.dbeaver.ext.mysql/plugin.xml

```diff
@@ -71,9 +71,9 @@
                                         </items>
                                     </items>
                                 </folder>
-                                <folder type="org.jkiss.dbeaver.ext.mysql.model.MySQLPartition" label="%tree.partitions.node.name" icon="#partitions" description="Table partitions" visibleIf="object.dataSource.supportsPartitions()">
+                                <folder type="org.jkiss.dbeaver.ext.mysql.model.MySQLPartition" label="%tree.partitions.node.name" icon="#partitions" description="Table partitions" visibleIf="object.dataSource.supportsPartitions()" virtual="true">
```

Comment:
Virtual nodes are supposed to be used for duplicated objects in a tree. How this helps in this case?